### DISCUSSION TITLE
How to add types file for javascript lib

### DISCUSSION THREAD

**Discussion by SettingDust** [denoland/deno#14902] [Q&A] [answered]
Context: https://github.com/gildas-lormeau/zip.js/issues/335

**Comment by nayeemrmn** [comment, marked as answer]
As you mentioned in https://github.com/gildas-lormeau/zip.js/issues/335#issuecomment-1159348236, adding `/// <reference types="./index.d.ts" />` to the top of `index.js` fixes typing on Deno, you should go ahead and contribute that change. Beyond that is a problem with the webstorm extension.
